ceph-build/build/build_mingw

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Should these files be in the `ceph-dev-new-build` directory instead?

Reply:
Yes.  The `ceph-build` directory is for upstream releases.